### DISCUSSION TITLE
OC 0.8.3 + 2013 / 2015 MacBook Pro SMBIOS (Stable Release)

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T440s#12] [Announcements]
![repository-open-graph-template new](https://user-images.githubusercontent.com/72415505/193061418-2920edc3-48f6-4397-b9da-e88fe475b626.png)

#
## macOS Monterey is now 100% stable! 💻 🖥

This release comes with two EFI folders, one that supports up to **macOS 11 Big Sur** and older which contains the 2013 13" MacBook Pro SMBIOS and one that supports **macOS 12 Monterey** which contains the 2015 13" MacBook Pro SMBIOS. The biggest change for this release is that the SMBIOS for macOS Monterey is finally **100% stable!**

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/releases/tag/OC-Stable'>OC 0.8.3 + 2013 / 2015 MacBook Pro SMBIOS (Stable Release)</a>.</em>

**Comment by elvisrv** [comment]
Just tested your EFI on a t440p Monterey 12.6.1 and apparently everything worked very well except for USB's. Do you have any ideia in what can I do to make it work? Tested with smbios MacBookPro11,4 and 12,1.

[Quick update]
Disabling USBInjectAll.kext did the trick for me. Now the USB's work. So far so amazing EFI.

**Reply by MultimediaLucario** [reply]
I'm glad you're enjoying it. I am currently in the works of getting macOS Ventura to work on the T440s.

**Reply by elvisrv** [reply]
Have you noticed any performance impact using smbios MacBookPro14,1?

**Reply by MultimediaLucario** [reply]
Nothing yet
